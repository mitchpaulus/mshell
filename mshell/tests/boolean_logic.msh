[echo [(true false and) (truey) (falsey)] if];
[echo [(true true and) (truey) (falsey)] if];
[echo [(false false and) (truey) (falsey)] if];
[echo [(true false or) (truey) (falsey)] if];
[echo [(true true or) (truey) (falsey)] if];
[echo [(false false or) (truey) (falsey)] if];

[echo];
[echo Nots];
[echo ========];
[echo [(true false and not) (truey) (falsey)] if];
[echo [(true true and not) (truey) (falsey)] if];
[echo [(false false and not) (truey) (falsey)] if];
[echo [(true false or not) (truey) (falsey)] if];
[echo [(true true or not) (truey) (falsey)] if];
[echo [(false false or not) (truey) (falsey)] if];
