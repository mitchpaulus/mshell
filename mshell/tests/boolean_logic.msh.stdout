falsey
truey
falsey
truey
truey
falsey

Nots
========
truey
falsey
truey
falsey
falsey
truey
